app/locale/de/LC_MESSAGES/django.po

```diff
@@ -155,7 +155,7 @@ msgstr "Vergleichen"
 
 #: core/menu.py:11
 msgid "candidates"
-msgstr "Kandidaten"
+msgstr "Kandidaten und Kandidatinnen"
```

Comment:
I think this is something you could add to the other languages too, since gendering is pretty important nowadays.